src/ui/templates/Welcome.jsx

```diff
@@ -59,39 +59,41 @@ function component({ version }) {
-                        architecture reduces previously existing major overheads and allows for some
-                        performance optimizations.
-                    </p>
+                    <h3>Exercise Decorations</h3>
                     <p>
-                        Unfortunately this change required for all exercises to be migrated to a new
-                        default location. If you would like to, you may now move them back again
-                        from the settings view.
+                        You can now see completed and partially completed (i.e. received some
+                        points) exercises with an icon on the course workspace.
+                        <br />
+                        We also implemented showing if the deadline has exceeded and if the exercise
+                        has been removed or renamed in the course. By hovering on an exercise with
+                        an icon, you should see an information message explaining the status.
                     </p>
+                    <img
+                        style="margin-bottom: 1em;"
+                        class="rounded mx-auto d-block"
+                        src={exerciseDecorations}
+                    />
                 </div>
                 <div class="content_section">
-                    <h3>Improved exercise downloads</h3>
+                    <h3>Migrated to VSCode Settings</h3>
                     <p>
-                        Following the change to exercise management, downloading new exercises is
-                        now significantly faster.
+                        We removed our own Settings webview and migrated the user and course
+                        specific settings to the VSCode Settings page. <br />
+                        With this change, users can now specify course specific settings when having
+                        the course workspace open in VSCode by going to Settings and selecting the
+                        Workspace tab. <br />
+                        Settings defined in the Workspace tab are of higher priority than those
+                        defined in the User scope. When adding a new course, it will copy the
+                        settings defined in the User scope to the Workspace.
                     </p>
                 </div>
                 <div class="content_section">
-                    <h3>Improvements to TMC workspace</h3>
+                    <h3>Automatically download old submissions</h3>
                     <p>
-                        TMC Workspace files have now been located in a consistent location separate
-                        from where exercises are. This allows for more user-friendly experience when
-                        moving the exercises folder or using the extension data wipe command.
+                        Since TMC VSCode version 2.0.0 we had to disable automatically downloading
```

Comment:
Prior to version 2.0.0, downloaded exercises were restored to the state of their latest submission.